<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>879</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefixBack/back.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Calibri&quot;;
border-image: url(:/newPrefixObj/1612493406_211-p-serii-tsvet-na-belom-fone-243.jpg);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Многоклассовая нейросеть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createModelButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>450</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>Построить модель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predictButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Предсказать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savecsvButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>760</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Сохранить .csv решение</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/1612493406_211-p-serii-tsvet-na-belom-fone-243.jpg);
font: 16pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Определение возраста человека по его лицу&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Точность:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Прогноз:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;double&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="prediction">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
font: 14pt &quot;Calibri&quot;;
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;age&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shootButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>Снимок</string>
    </property>
   </widget>
   <widget class="QLabel" name="val_accuracy">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;Точность&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openImageButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Открыть изображение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;val_accuracy = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="saveAnswer">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>810</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/20bc56977efc63a66c8c8ed2413875ec.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphicsButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>590</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Показать графики</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>760</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/myPrefix/unnamed.jpg);
border-image: url(:/newPrefix/unnamed.jpg);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="infoAboutVGG16Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Информация о VGG-16</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/newPrefixObj/sunlight-white-simple-background-minimalism-sky-text-blue-gradient-texture-circle-atmosphere-gray-empty-light-color-fog-shape-line-screenshot-atmospheric-phenomenon-computer-wallpaper-atmosphere-of-earth-238647.jpg);
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.44 rgba(252, 128, 128, 80), stop:0.512438 rgba(124, 112, 118, 156), stop:0.751244 rgba(255, 188, 188, 80), stop:0.756219 rgba(255, 162, 162, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Predict below&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Файлы</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionWebCam"/>
   </widget>
   <addaction name="menuOpen"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Open&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionWebCam">
   <property name="text">
    <string>WebCam</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="diplom.qrc"/>
 </resources>
 <connections/>
</ui>
